package.json

```diff
@@ -11,14 +11,16 @@
     "build:lib": "babel --out-dir lib src",
     "prebuild:umd": "rm -rf dist/*",
     "prebuild:dist": "rm -rf dist/*",
+    "build:dev": "rollup -c",
```

Comment:
I'm not sure we need a separate command – with this example, I'd expect to be able to switch out `styled-components.js` with `styled-components.min.js` in the `index.html` and to test my changes in minfied mode, right?